src/renderer/families/tron/AccountHeaderActions.js

```diff
@@ -28,8 +30,14 @@ type Props = {
 
 const AccountHeaderActions = ({ account, parentAccount }: Props) => {
   const dispatch = useDispatch();
-  const { tronResources } = account;
+  const unit = getAccountUnit(account);
+  /** min 1TRX transactions */
+  const minAmount = 10 ** unit.magnitude;
```

Comment:
⚠️  if you change the account unit. it will not be 1 TRX !

moreover, I strongly recommend to hardcode this in a constant (outside of the component) – as well as any other usage of this.

it should probably be:

```js
const minAccount = BigNumber(10).pow(getCryptoCurrencyById("tron").units[0].magnitude)
```

Reply:
Ok i'll make a PR for this :+1: